new_platforms/vscode-extension/README.md

```diff
@@ -0,0 +1,110 @@
+# Open Enclave extension for Visual Studio Code
+
+A [Visual Studio Code](https://code.visualstudio.com/) [extension](https://marketplace.visualstudio.com/VSCode) supporting [Open Enclave](https://github.com/Microsoft/openenclave), including development, debugging, emulators, and deployment!
+
+## Getting started
+
+You can install the Open Enclave extension into Visual Studio Code directly via the Visual Studio Gallery [TBD](TBD).
+
+Alternatively, you can run the extension from this repository by following these instructions:
+
+1. Clone this repository `git clone --recursive https://github.com/microsoft/openenclave`
+2. Navigate to `new_platforms\vscode-extension` in the cloned folder.
+3. Run npm to install the dependencies: `npm install`
+4. Start VSCode: `code .`
+5. Start the extension using F5
+
+## Features
+
+### Create a new Open Enclave solution.
+
+You can use the **Microsoft Open Enclave: New Open Enclave Solution** command to create a new 
+Open Enclave solution.  
+
+You will be prompted to:
+
+1. Select a folder for your solution.  
+1. On Linux, you will have the option to create a standalone project or an Azure IoT Edge container 
+    project.  Otherwise, you will only have the option to create an Azure IoT Edge container project.
+1. If you choose a container project, you will be prompted to provide your container repository.
```

Comment:
Probably need a link to somewhere that talks about creating a container repository.

Reply:
ok